<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSCAppWidget</class>
 <widget class="QWidget" name="MSCAppWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="horizontalSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftArea" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="msc::DocumentTreeView" name="documentTree"/>
       </item>
       <item>
        <widget class="QPushButton" name="ivSwitch">
         <property name="toolTip">
          <string>Open the file</string>
         </property>
         <property name="text">
          <string>Interface view</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="asn1Widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="asn1Switch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Open the file</string>
            </property>
            <property name="text">
             <string>file.asn</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="asn1Select">
            <property name="toolTip">
             <string>Select the ASN.1 file</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="asn1::ASN1FileView" name="asn1View" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="editorsWidgets" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="shared::ActionsBar" name="documentToolBar" native="true"/>
       </item>
       <item>
        <widget class="shared::ActionsBar" name="chartToolBar" native="true"/>
       </item>
       <item>
        <widget class="QStackedWidget" name="centerView">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="msc::GraphicsView" name="chartView"/>
         <widget class="msc::GraphicsView" name="hierarchyView"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>shared::ActionsBar</class>
   <extends>QWidget</extends>
   <header>actionsbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>msc::GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>msc::DocumentTreeView</class>
   <extends>QTreeView</extends>
   <header>documenttree/documenttreeview.h</header>
  </customwidget>
  <customwidget>
   <class>asn1::ASN1FileView</class>
   <extends>QWidget</extends>
   <header>asn1fileview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
